.flowconfig

```diff
@@ -1,5 +1,7 @@
 [ignore]
-.*/example/index.js
+<PROJECT_ROOT>/example/
```

Comment:
:+1: Nice! I was looking for something similar and couldn't find `<PROJECT_ROOT>` in the docs.